### DISCUSSION TITLE
parsing a large json document without loading it completely in memory

### DISCUSSION THREAD

**Discussion by iscu2020** [networknt/json-schema-validator#1102] [Q&A] [answered]
Hello,
I tried to find such a functionality in the current implementation but I wasn't successful. 

Basically we need to parse large json files using a schema. Because of that, loading the entire json string in memory is not an option.

Could anybody tell if we can use a input stream instead ?

**Comment by justin-tay** [comment, marked as answer]
This is essentially the same question as 
* https://github.com/networknt/json-schema-validator/discussions/995

This library does not support streaming the input.
